src/nidigital/metadata/functions.py

```diff
@@ -2860,38 +3847,56 @@
     'WriteSourceWaveformSiteUniqueU32': {
```

Comment:
Copy documentation from this function to the fancy version of this function:
https://github.com/ni/nimi-python/blob/30687660fc0bc57b32d6b9639d95f55a52a7a312/src/nidigital/metadata/functions_addon.py#L82

Reply:
Done

Reply:
I still see TBD in generated docstring for `write_source_waveform_site_unique`?